clic/include/tier1.hpp

```diff
@@ -47,8 +47,8 @@ add_images_weighted_func(const Device::Pointer & device,
                          const Array::Pointer &  src0,
                          const Array::Pointer &  src1,
                          Array::Pointer          dst,
-                         float                   factor0,
-                         float                   factor1) -> Array::Pointer;
+                         float                   factor1,
+                         float                   factor2) -> Array::Pointer;
```

Comment:
I know it is to fit the `prototype` but the input index being 0 and 1, and the factor index being 1 and 2 triggers my OCD.

Wouldn't we want to also use this effort to correct possible wrong doing in the API developpment during the `prototype` ( i know I am annoying :wink: )

Reply:
The documentation is also commonly "First image to process"